app/assets/javascripts/departments.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I've been deleting these coffee files; I don't think we use coffeescript anywhere else?